src/elements/content-sidebar/activity-feed/activity-feed/ActivityFeed.js

```diff
@@ -177,9 +202,16 @@ class ActivityFeed extends React.Component<Props, State> {
                     }}
                     className="bcs-activity-feed-items-container"
                 >
-                    {isEmpty ? (
-                        <EmptyState isLoading={isLoading} showCommentMessage={showCommentForm} />
-                    ) : (
+                    {isLoading && (
+                        <div className="bcs-activity-feed-loading-state">
```

Comment:
Time for SUIT?

Reply:
If you don't mind, I'd prefer to make that change separately. It's likely to be pretty large. Unless you mean changing this one selector to SUIT?

Reply:
I don't mind at all. It's more a statement that, if we're going to be mucking around in here more now, we should take the time to convert it.